### PR TITLE
Admin enables/disables a user account

### PR DESCRIPTION
**Pull request by rer7891**
## What functionality does this accomplish?
closes # 59, 60

**Description:**
Admin can enable/disable user account. User can no longer log in and will not be counted towards statistics once disabled.
### Test Coverage Percentage: 100%
- [ ] No Tests have been changed
- [x] Some Tests have been changed - adds testing for this functionality
- [ ] All of the Tests have been changed(Please describe what in the world happened):
- [ ] New Tests added for new features

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Helpful Resources:
* Resource link AND small description of info gathered/learned

## Please include a gif of how you feel about this branch:
giphy

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -1,4 +1,6 @@
 class User <ApplicationRecord
+
+  scope :active, -> { where(active: 'true') }
```
Comment (jfangonilo) [inline app/models/user.rb]: Nice use of scoping

File: app/controllers/admin/users_controller.rb [hunk 2]
```diff
@@ -1,9 +1,20 @@
 class Admin::UsersController < Admin::BaseController
   def index
-    @users = User.all
+    @users = User.unscoped.all
```
Comment (wrenisit) [inline app/controllers/admin/users_controller.rb]: Great use of scope

File: app/controllers/sessions_controller.rb [hunk 3]
```diff
@@ -19,7 +19,8 @@ def create
       session[:user_id] = user.id
       welcome(user)
     else
-      flash[:error] = "Sorry Invalid Password or Email."
+      flash[:error] = "Sorry Invalid Password or Email." if user.active
+      flash[:error] = "Sorry This Acount Is Inactive" if !user.active
```
Comment (wrenisit) [inline app/controllers/sessions_controller.rb]: Elegant solution. I like that you one lined it.

File: app/views/admin/users/index.html.erb [hunk 4]
```diff
@@ -9,6 +10,12 @@
 <p>Address:</p>
 <p><%= user.address %></p>
 <p><%= user.city %>, <%= user.state %> <%= user.zip %></p>
+<% if user.active %>
+  <%= link_to "Deactivate User", "/admin/users/#{user.id}", method: :patch %>
```
Comment (wrenisit) [inline app/views/admin/users/index.html.erb]: Great code!

**Pull request merged**